appside/frames/consent_disclosure.js

```diff
@@ -0,0 +1,137 @@
+"use strict";
+
+/**
+ * Rendering (View) code for consent disclosure frame
+ * @author Rachel Sitt
+ */
+
+
+/**
+ * A frame specifically purposed for consent disclosure.
+ * 
+ * Frame contains pdf file and a set of checkboxes.
+ **/
+class ConsentDisclosureFrame extends Frame {
+
+    /**
+     * Construct ConsentDisclosureFrame from an object
+     * 
+     * @param frame_data -- Object containing the frame's data. Expected fields:
+     *    frame_data.template -- The frame's template
+     *    frame_data.title (string) -- The frame's title
+     *    frame_data.instructions (string) -- instructions for user to read pdf
+     *    frame_data.questions (Array of map of string) -- 2 key/value pairs
```

Comment:
I think "2" is inaccurate here. Let's leave it open for any number of consent questions.

Reply:
Fixed.